src/ThemedControls.cpp

```
@@ -20,6 +24,246 @@ wxColour Blend(const wxColour& a, const wxColour& b, double t) {
+  }
+
+  void EnsureVisible(int i) {
+    const int y = i * m_row_h;
+    const int ch = GetClientSize().y;
+    if (y < m_scroll)
+      m_scroll = y;
+    else if (y + m_row_h > m_scroll + ch)
+      m_scroll = y + m_row_h - ch;
+  }
+
+  void Pick(int i) {
```

Clang (14.0.6)

[warning] method 'Pick' can be made static

(readability-convert-member-functions-to-static)